data/pigui/modules/info-view/06-flightlog.lua

```diff
@@ -236,48 +159,11 @@ local function checkbox(label, checked, tooltip)
 	return changed, ret
 end
 
-local function exportLogs()
-	-- TODO localize?
-	local foldername = FileSystem.MakeUserDataDirectory( "player_logs" )
-
-    local player = Character.persistent.player
-
-	local base_save_name = player.name
-
-	local formatter
-	local extension
-	if export_html then
-		formatter = html_formatter
-		extension = '.html'
-	else
-		formatter = text_formatter
-		extension = '.log'
-	end
-
-	local log_filename = FileSystem.JoinPath( foldername, base_save_name .. extension )
-
-	formatter:open( io.open( log_filename, "w" ) )
-
-	for entry in FlightLog:GetLogEntries(getIncludedSet(),nil, earliest_first) do
-
-		writeLogEntry(entry, formatter, true)
-
-		if (entry:HasEntry()) then
-			formatter:headerText(l.ENTRY, entry:GetEntry(), true)
-		end
-		formatter:separator()
-	end
-
-	formatter:close()
-
-end
-
 local function displayFilterOptions()
 	ui.spacing()
 	local c
 	local flight_log = true;
 
-	c,include_player_info = checkbox(l.PERSONAL_INFORMATION, include_player_info)
 	c,include_custom_log = checkbox(l.LOG_CUSTOM, include_custom_log)
 	c,include_station_log = checkbox(l.LOG_STATION, include_station_log)
 	c,include_system_log = checkbox(l.LOG_SYSTEM, include_system_log)
```

Comment:
As a note, if `include_custom_log` isn't used significantly anywhere else this could become:
```lua
c, includedEntries["Custom"] = checkbox(l.LOG_CUSTOM, includedEntries["Custom"])
-- ...
```

This might allow dropping the per-frame new table allocation in `getIncludedSet`.